src/extension.ts

```diff
@@ -1098,6 +1098,8 @@ class ExtensionManager implements vscode.Disposable {
       const cmtFolder = this._folders.get(workspaceFolder);
       if (cmtFolder) {
         return fn(cmtFolder.cmakeTools);
+      } else {
+        rollbar.error(localize('no.active.folder', 'No active folder.'));
```

Comment:
Is it an error to not have an active folder?  I'm not sure the user can do anything about it, so it's probably best not to bug them about it.

Reply:
before fixing the problem, this was where the error was happening (i.e. in `this._folders.get(workspaceFolder);`). I guess the error message is not necessary anymore as the error is prevented now.